test/unit/v1/test_bucket.py

```diff
@@ -241,6 +241,30 @@ def _make_file(self, file_id, file_name):
         return self.bucket.start_large_file(file_name, 'text/plain', {})
 
 
+class TestGetFileInfo(TestCaseWithBucket):
```

Comment:
Do the same for `v0`

Reply:
Why would we do that? We're not backporting this to v0 are we?

Reply:
It's backported automatically. This is how ApiVer works (the thing that is used here).
In short: `v0` imports everything from `v1` and if we need to preserve the old behavior in v0, we create wrappers there.
@ppolewicz can say more about that ;)